src/main/kotlin/net/maizegenetics/phgv2/utils/SeqUtils.kt

```diff
@@ -43,10 +43,39 @@ private val myLogger = LogManager.getLogger("net.maizegenetics.phgv2.utils.SeqUt
  *      e.g. LineA:1,2
  *           LineB:1,2
  *
+ * 4. retrieveRefSampleName: called to get the name of the sample used as the reference in the AGC file.
+ *    Expected user input: dbPath:String
+ *    THe command sent to age is "agc listref <in.agc>"  THe output is written to the output stream
+ *    and returned from the function as a string.
+ *
  * The rest of the functions in this file are used internally by the above functions.
  *
  */
 
+/**
+ * Retrieves the name of the sample used as the reference in the AGC file.
+ * @param dbPath    the folder containing assemblies.agc
+ * @return the name of the sample used as the reference in the AGC file
+ */
+fun retrieveRefSampleName(dbPath: String,condaEnvPrefix:String):String {
```

Comment:
You added this function but it's not used anywhere?

Reply:
oh - right.  I was debugging in the gvcf_from_hvcf branch, then copied this file over.  That branch uses the retrieveRefSampleName() method.  I'll remove it from this pull request.